cluster/manifests/skipper/deployment.yaml

```diff
@@ -136,6 +136,7 @@ spec:
           - "-metrics-flavour=prometheus"
           - "-enable-connection-metrics"
           - "-enable-route-lifo-metrics"
```

Comment:
Are we keeping the `lifo` metrics on purpose?

Reply:
Keeping it until the rollout of fifo is complete.

Reply:
Yes as long as we have lifo somewhere